discover/portal/articles/230-calendar/01-using-the-calendar-widget.markdown

```diff
@@ -0,0 +1,130 @@
+a colored box: click on it to show/hide that calendars events in the main
+viewing area.
+
+## Adding New Calendars [](id=adding-new-calendars)
+
+Follow these steps to create a new personal calendar:
+
+1.  Click on the arrow to the right of the *My Calendars* header and select
+    *Add Calendar* from the menu. 
+
+2.  Fill in the *Add Calendar* form. Give the calendar a name and description,
+    set a time zone, and determine if it will be your user's *default
+    calendar*---the one that is shown automatically whenever the widget is
+    displayed. You can also pick a color, which will color code events whenever
+    multiple calendar's events are displayed at once. You can also decide
+    whether to enable ratings or comments on the calendar's events, and
+    configure permissions.
+
+To edit an existing calendar instead of adding a new one, select *Manage
+Calendars* from the menu in step 1.
+
+To add or edit a site calendar, open the menu next to the header with the
+site's name instead of *My Calendars*.
+
+![Figure 3: Personal and site calendars are shown in the lower left. This image shows calendars belonging to user *Test Test* and site *Liferay DXP*.](../../images/new-calendar-manage-calendars.png)
+
+## Adding Events to a Calendar [](id=adding-events-to-a-calendar)
+
+Follow these steps:
+
+1.  Click on any day in the main viewing area to open an event creation pop up.
+    If you've selected the *Day* or *Week* view, you can click on the specific
+    time when your event begins.
+
+    ![Figure 3: When you click anywhere on the calendar, you'll see the event creation pop up appear. Click *Edit* to specify details for your event.](../../images/new-calendar-event-popup.png)
+
+2.  Name your event and select and assign it to a calendar. Click *Save* to
+    create the event immediately or *Edit* to enter additional information.
+
+    ![Figure 4: You can specify event details such as the event title, start date, end date, description, location, and more.](../../images/new-calendar-event-details.png)
+
+3.  If clicked *Edit*, fill in the *Edit Event* form. Enter a start and end
+    time and enter a description. To schedule an event that will reoccur
+    regularly, check the *Repeat* box and fill in the *Repeat* pop up.
+
+    ![Figure 5: The *Repeat* box allows you to specify whether an events repeats daily, weekly, monthly, or yearly, how often it repeats, and when (or if) it ends.](../../images/new-calendar-event-repeat.png)
+
+### Additional Event Functions
+
+At the bottom of the *Edit Event* form, there are several collapsed sections:
+*Details*, *Invitations*, *Reminders*, *Categorization*, and related *Related
+Assets*.
+
+#### Details
+
+In the Details collapsible section, you can change the calendar to which the
+event belongs, enter a location, and configure the event's permissions.
+
+Giving a user permission to add, delete, or update discussion allows them to
```

Comment:
I've checked this section on master and I've not found the permissions configuration.